test/nightly/azureResourceSQL.test.ts

```diff
@@ -35,4 +35,16 @@ suite('SQL action', async function (this: ISuiteCallbackContext): Promise<void>
         const getAccount: CosmosDBManagementModels.DatabaseAccount | undefined = await client.databaseAccounts.get(resourceGroupName, accountName);
         assert.ok(getAccount);
     });
+
+    test('Delete account', async () => {
```

Comment:
Could you update this test and the Mongo test to say SQL account/ Mongo account?